CHANGELOG.md

```diff
@@ -2,38 +2,56 @@
+
+## ☢️ Upgrade Warnings ☢️
+- The introduction of the new `ChainIndexer` subsystem replaces the existing `MsgIndex`, `EthTxHashLookup`, and `EventIndex` implementations. This change may affect RPC providers and node operators who use or expose Ethereum and/or events APIs. It is crucial to review the [ChainIndexer documentation for operators](./documentation/en/chain-indexer-overview-for-operators.md) to understand how to enable, configure, and use the new Indexer.
+
+## ⭐ New Feature Highlights:
+- New ChainIndexer subsystem to index Filecoin chain state such as tipsets, messages, events and ETH transactions for accurate and faster RPC responses. The `ChainIndexer` replaces the existing `MsgIndex`, `EthTxHashLookup` and `EventIndex` implementations in Lotus, which [suffer from a multitude of known problems](https://github.com/filecoin-project/lotus/issues/12293).  If you are an RPC provider or a node operator who uses or exposes Ethereum and/or events APIs, please refer to the [ChainIndexer documentation for operators](./documentation/en/chain-indexer-overview-for-operators.md) for information on how to enable, configure and use the new Indexer.  While there is no automated data migration and one can upgrade and downgrade without backups, there are manual steps that need to be taken to backfill data when upgrading to this Lotus version, or downgrading to the previous version without ChainIndexer. Please be aware that that this feature removes some options in the Lotus configuration file, if these have been set, Lotus will report an error when starting. See the documentation for more information
 - `lotus chain head` now supports a `--height` flag to print just the epoch number of the current chain head ([filecoin-project/lotus#12609](https://github.com/filecoin-project/lotus/pull/12609))
-- `lotus-shed indexes inspect-indexes` now performs a comprehensive comparison of the event index data for each message by comparing the AMT root CID from the message receipt with the root of a reconstructed AMT. Previously `inspect-indexes` simply compared event counts, comparing AMT roots confirms all the event data is byte-perfect. ([filecoin-project/lotus#12570](https://github.com/filecoin-project/lotus/pull/12570))
-- Expose APIs to list the miner IDs that are currently participating in F3 via node. ([filecoin-project/lotus#12608](https://github.com/filecoin-project/lotus/pull/12608))
-- Implement new `lotus f3` CLI commands to list F3 participants, dump manifest, get/list finality certificates and check the F3 status. ([filecoin-project/lotus#12617](https://github.com/filecoin-project/lotus/pull/12617), [filecoin-project/lotus#12627](https://github.com/filecoin-project/lotus/pull/12627))
-- Return a `"data"` field on the `"error"` returned from RPC when `eth_call` and `eth_estimateGas` APIs encounter `execution reverted` errors. ([filecoin-project/lotus#12553](https://github.com/filecoin-project/lotus/pull/12553))
```

Comment:
@rjan90 what's the criteria for what makes it into the changelog and what's dropped? I went to the changelog today to see whether this particular item was included in 1.31 so I could point someone to it (for https://github.com/filecoin-project/lotus/issues/12737) but it's not there so I had to go `git log` digging to figure it out. Same with the `lotus f3` one above it which has been dropped. Is it just a matter of picking highlights and removing the cruft that's not worth highlighting? I imagined that the `skip/changelog` tag mostly helped us with that at time of merge. But maybe I need to be paying more attention to these PRs.

On the flip side, `lotus-shed indexes inspect-indexes` is now irrelevant because the file that was introduced into has been deleted because of ChainIndexer .. ooops, we probably should have dropped that line in the changelog, or just not backported.